spec/jobs/wild_collection_job_spec.rb

```diff
@@ -4,7 +4,7 @@
   let(:filename) { "example_of_wild_collection_data.csv" }
 
   before do
-    Rails.application.load_seed if Facility.valid_codes.empty?
+    Rails.application.load_seed
```

Comment:
I'm not sure if this is the right approach, for some reason, the cache is already loaded with some Facilities, but not all are present, causing the tests to fail

Reply:
Does `load_seed` do any kind of dupe suppression? would we likely run into any problems with possibly loading db seeds twice?

Reply:
That I don't know, will have to look into it.

Reply:
Looking at the doc, it should be ok: https://apidock.com/rails/ActiveRecord/Relation/load

But also tested it out like so:
```
  before do
    Rails.application.load_seed
  end

  before(:each) do
    Rails.application.load_seed
  end
```

And it doesn't dupe the records

Reply:
Thank you for checking! :heavy_check_mark::heavy_check_mark:

Reply:
I was mistaking and thought `before` defaults to `default(:all)`. 

Tested it like so:
```
before(:all) do
...
end

before(:each) do
...
end
```

Result remains the same. 👍

Reply:
As a note, :all does NOT automatically rollback any changes in the DB. I think that's fine here, though. Just including this comment for completeness 🎈

Reply:
Oh, good to know. I haven't ran into any issues yet it not being rolled back automatically. Will keep that in mind.